neuralmagic/tests/test_nm-vllm_licenses.py

```diff
@@ -0,0 +1,146 @@
+import re
+from importlib.metadata import metadata
+from pathlib import Path
+from typing import Tuple
+
+from pytest import mark, param
+
+import vllm
+
+THIRD_PARTY_LICENSE_FILES = [
```

Comment:
this looks okay for now. i'm wondering if there is someway to tie this into "setup.py". i'm worried the list here can get out of sync.

Reply:
It will get out of sync, but there's a test that will fail if a new license is added to the `licenses` directory, basically alerting us to (1) confirm that the new license is required,  and (2) update this list.  That will only leave us with the scenario where some new code is being used and we *should* have a license, but we don't.  as far as I can tell, we're dependent on a person figuring that out.

Reply:
cool